resources/qml/dialogs/InviteDialog.qml

```diff
@@ -83,85 +121,99 @@ ApplicationWindow {
                         cleanUpAndClose();
 
                 }
+                onTextChanged: {
+                    searchTimer.restart()
+                    if(isValidMxid) {
+                        profile = TimelineManager.getGlobalUserProfile(text);
+                    } else
+                        profile = null;
+                }
+                Timer {
+                    id: searchTimer
+
+                    interval: 350
+                    onTriggered: {
+                        userSearch.model.setSearchString(parent.text)
+                    }
+                }
             }
 
-            Button {
```

Comment:
Why did you remove this button? :3

Reply:
Users are now added by clicking on them, not by clicking on the "add" button. It makes more sense if there's more than one user to choose from.

apparently I introduced the bug when I tried to call beginInsertRows() before making changes... fixed in the upcoming commit.

Reply:
Ah, I guess that makes sense. It wasn't that obvious to me, since I am used to the old design though, but maybe it is actually more intuitive.